### PR TITLE
feat: Create Getting Started layout

### PR DESCRIPTION
**Pull request by skamath2000**
Contributes to #1515.
 
Creates a new GettingStartedModule designed by Sarah Pedregon (including style, code, documentation, images).
Creates an example Getting Started Layout that uses the GettingStartedModule.
This PR was previously opened on Carbon for IBM Security repo but is moved to this repo and updated to use the new PageHeader from IBM Cloud and Cognitive.

#### What did you change?
- Added new GettingStartedModule component
- Added new Getting Started Page Layout
- Removed Grid from Page Layout decorator and included Grid in layouts

#### How did you test and verify your work?
`yarn test` was run. Verified the changes on the development server.

Looking for any feedback on technical approach.

### CODE REVIEW

**Review by lee-chase — APPROVED** [review]

**Comment by lee-chase** [comment]
You'll need to update the style snapshot @skamath2000. You can do this by adding '-u' on the end of the test command.

**Comment by elycheea** [comment]
@SimonFinney @dbrugger Can one of you provide a review on this one?

**Comment by skamath2000** [comment]
> You'll need to update the style snapshot @skamath2000. You can do this by adding '-u' on the end of the test command.

Updated!

**Comment by skamath2000** [comment]
@dcwarwick Updated snapshots here as well.

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import github from './images/github.png';
+import slack from './images/slack.png';
+
+### Troubleshooting
+
+Visit our [GitHub](https://github.com/carbon-design-system/ibm-security) to view
```
Comment (jlongshore) [inline packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx:6]: this link should point to https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/security

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+and submit issues, or reach out on
+[Slack](https://app.slack.com/client/T1V6D3RT7/CRV39QKP0) for additional
```
Comment (jlongshore) [inline packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx:8]: This is pointing to the slack channel for the deprecated repo

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+support.
+
+<Row>
+  <Column>
+    <a
+      className="storybook__link--image"
+      href="https://github.com/carbon-design-system/ibm-security"
```
Comment (jlongshore) [inline packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx:15]: https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/security

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      <img alt="Troubleshooting GitHub link" src={github} />
+    </a>
+  </Column>
+
+  <Column>
+    <a className="storybook__link--image" href="https://app.slack.com/client/T1V6D3RT7/CRV39QKP0" rel="noopener noreferrer" target="_blank">
```
Comment (jlongshore) [inline packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx:24]: slack channel for deprecated repo, assuming something like https://ibm-security.slack.com/archives/CQGR0HC05

**Review by jlongshore — CHANGES_REQUESTED** [review]
Looks good mostly, just some links pointing to deprecated repos / slack channels for that deprecated repo...

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx (hunk 4 above)
Comment (amtoussam) [reply packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx:25]: @jlongshore I believe this is good. lmk

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx (hunk 3 above)
Comment (amtoussam) [reply packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx]: @jlongshore I believe this is good. lmk

File: packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx (hunk 2 above)
Comment (amtoussam) [reply packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx]: @jlongshore I believe this is good. lmk

**Review by jlongshore — APPROVED** [review]

**Pull request merged**